prog1.py

```diff
@@ -0,0 +1,28 @@
+import psutil
+import subprocess
+import re
+import time
+
+file_num = 3
+file_size = 4096
+#free_space = 
+
+
+# Detect local mounted disk
+local_disk_mounts = []
+partitions = psutil.disk_partitions(all=False)
+pattern = re.compile(".*/dev/[h,s]d[a-z].*")
+for p in partitions:
+   if pattern.match(str(p)) and psutil.disk_usage(p.mountpoint).free > 1024:
```

Comment:
`1024` value there should be the `X` parameter: 
>  Detect local mounted disk (make sure it is local) with at least X MB free space

Next, what is the unit of space returned by `.free` properly: KB, MB, bytes?